docs/tips.md

```diff
@@ -210,6 +234,26 @@ run = '''
 
 Credits to [@aidanzhai for sharing it](https://t.me/yazi_rs/3325/15373) in Yazi's telegram group.
 
+## Browse hovered file or directory in Git repository on Github {#browse-on-github}
+
+```toml
+[[manager.prepend_keymap]]
+on = [ "g", "b"]
+run = '''
+    shell 'gh browse .'
+desc = "Browse cwd on Github."
+'''
+
+[[manager.prepend_keymap]]
+on = [ "g", "f"]
+run = '''
+    shell 'gh browse $(git ls-files $0) --branch=$(git branch --show-current)'
```

Comment:
Why `$(git ls-files $0)` and ` --branch=$(git branch --show-current)` is needed here?

Reply:
Well if you just do `gh browse` wherever you are in the repo, it will open the `https://github.com/name/repo_name` instead of location of file.
And branch is required since if not mentioned, it will always go to main/master branch even if you are in some other branch.
That's why the above keybinding will simply open the exact location of the file. Hope its clear :)

Reply:
now that you mention it, I think you should have branch for the 2nd keybinding I mentioned. 
But it would be create to combine both of them into 1, since "gf" doesnt work for hovered directory.